### PR TITLE
improve  Password verification rules

### PR DESCRIPTION
**Pull request by TianWuwt**
## Summary

improve Password verification rules 
image

### CODE REVIEW

File: ui/src/components/customModal/ResetPwdModal.tsx [hunk 1]
```diff
@@ -108,6 +108,57 @@ export default function ResetPwdModal({
                 defaultMessage: '请输入',
               }),
             },
+            () => ({
```
Comment (powerfooI) [inline ui/src/components/customModal/ResetPwdModal.tsx:111]: Why not use `passwordRules` here?

Comment (TianWuwt) [reply ui/src/components/customModal/ResetPwdModal.tsx:111]: There are many original rules and may not be compatible

Comment (powerfooI) [reply ui/src/components/customModal/ResetPwdModal.tsx:111]: `passwordRules` are actually a subset of this list. Use `...passwordRules` here for reusing.

File: ui/src/pages/Cluster/New/helper.ts [hunk 2]
```diff
@@ -41,65 +41,4 @@ function generateRandomPassword() {
...
-            defaultMessage: '大小写字母、数字和特殊字符都至少包含 2 个',
-          }),
-        ),
-      );
-    },
-  }),
-];
-
-export { generateRandomPassword, passwordRules};
+export { generateRandomPassword};
```
Comment (powerfooI) [inline ui/src/pages/Cluster/New/helper.ts:44]: This seems not to be formatted.

**Review by powerfooI — APPROVED** [review]
LGTM

File: ui/src/components/customModal/ResetPwdModal.tsx (hunk 1 above)
Comment (TianWuwt) [reply ui/src/components/customModal/ResetPwdModal.tsx]: done

File: ui/src/pages/Cluster/New/helper.ts (hunk 2 above)
Comment (TianWuwt) [reply ui/src/pages/Cluster/New/helper.ts]: done

**Pull request merged**